plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -163,11 +178,11 @@ void startDevServer(List<File> runnables, MessageConsoleStream console, String h
    *
    * @param runnables the path to directories that contain configuration files like appengine-web.xml
    * @param console the stream (Eclipse console) to send development server process output to
-   * @param host the host name to which application modules should bind
+   * @param port the app service port
```

Comment:
Have the ports on the `LocalAppEngineServerBehaviour` instance.  It represents the actual running instance, right?  Certainly we're holding on to other state (the dev process, the `AppEngineDevServer`)